flink-runtime/src/test/java/org/apache/flink/runtime/zookeeper/ZooKeeperResource.java

```diff
@@ -48,7 +49,7 @@ private void verifyIsRunning() {
     @Override
     protected void before() throws Throwable {
         terminateZooKeeperServer();
-        zooKeeperServer = new TestingServer(true);
+        zooKeeperServer = ZooKeeperTestUtils.createAndStartZookeeperTestingServer();
```

Comment:
I overlooked the `zooKeeperServer.stop()` method here. We could change to `close()` as well